ui/app/components/app-item/deployment.hbs

```diff
@@ -22,9 +22,9 @@
     </small>
   </LinkTo>
   {{#if (and @deployment.preload.deployUrl (not-eq @deployment.state 4))}}
-    <ExternalLink href="https://{{@deployment.preload.deployUrl}}" class="button button--secondary">
+    <ExternalLink href="https://{{@deployment.preload.deployUrl}}" class="button button--secondary button--external-link">
```

Comment:
As a future change, perhaps we should make `<ExternalLink>` apply `button--external-link` automatically 🤔